src/passes/stringify-walker-impl.h

```diff
@@ -31,26 +31,15 @@ inline void StringifyWalker<SubType>::doWalkModule(Module* module) {
 
 template<typename SubType>
 inline void StringifyWalker<SubType>::doWalkFunction(Function* func) {
+  addUniqueSymbol(SeparatorCtx::makeFuncStart(func));
```

Comment:
Should we move this into `walk` as well so it is closer to its closing partner? Do we even need a custom implementation of `doWalkFunction` after that?

Reply:
after talking with you, managed to get the start/end separators near each other :) done